sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -185,9 +241,9 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     static SetAccessControlRecursiveResponse CreateFromJson(const nlohmann::json& node)
     {
       SetAccessControlRecursiveResponse result;
-      result.DirectoriesSuccessful = std::stoi(node["directoriesSuccessful"].get<std::string>());
-      result.FilesSuccessful = std::stoi(node["filesSuccessful"].get<std::string>());
-      result.FailureCount = std::stoi(node["failureCount"].get<std::string>());
+      result.DirectoriesSuccessful = node["directoriesSuccessful"].get<std::int32_t>();
```

Comment:
Be consistent. I've seen lots of `int32_t` and `int64_t`.

Reply:
Only length related is int64_t. Which one are you referring to?